### PR TITLE
Improve docs on E2E tests / update .env samples

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: docs/tests.md [hunk 1]
```diff
@@ -80,14 +57,14 @@ To get debug output when running cypress:
 ### Run E2E tests against local containers
 All dependencies will be mocked, including upstream APIs used by make-recall-decision-api, and HMPPS Auth.
 
-Set the CYPRESS_USERNAME, CYPRESS_PASSWORD, CYPRESS_USERNAME_SPO and CYPRESS_PASSWORD_SPO env vars in the [.env.sample](./.env.sample) file when you copy it to .env. You can obtain the username and password from the team.
+Set the CYPRESS_PASSWORDO and CYPRESS_PASSWORD_SPO env vars in the [.env.local.sample](./.env.local.sample) file when you copy it to .env. The passwords can be obtained from the CYPRESS_PASSWORD_local and CYPRESS_PASSWORD_SPO_local env vars in [CircleCi](https://app.circleci.com/settings/project/github/ministryofjustice/make-recall-decision-ui/environment-variables)
```
Comment (jackmaloney) [inline docs/tests.md:60]: Typo in `CYPRESS_PASSWORDO`

File: .env.dev.sample [hunk 2]
```diff
@@ -9,7 +9,7 @@ TOKEN_VERIFICATION_API_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/veri
 API_CLIENT_ID=<user_name>
 API_CLIENT_SECRET=<client_secret>
 SESSION_SECRET=session-secret
-CYPRESS_USERNAME=MAKE_RECALL_DECISION_USER
```
Comment (jackmaloney) [inline .env.dev.sample:12]: How come you've removed these from the sample?

File: docs/tests.md [hunk 3]
```diff
@@ -97,16 +74,29 @@ Open Cypress, from there you can run the tests:
...
-### Run E2E tests against dev
+### Running E2E tests against dev or preprod env on CircleCI
+
+If the tests in the CI pipeline fail, you can try to reproduce by running the E2E tests in your local repo against dev or preprod env.
+
+You can run your local tests against dev env using:
+
 ```
-npx cypress open --env USERNAME=<username>,PASSWORD=<password>,USERNAME_SPO=<username_spo>,PASSWORD_SPO=<password_spo>,CRN=X098092,CRN2=X514364,ENV=dev --config-file e2e_tests/cypress.config.ts --config baseUrl=https://make-recall-decision-dev.hmpps.service.justice.gov.uk
+npx cypress open --env USERNAME=<username>,PASSWORD=<password>,USERNAME_SPO=<username_spo>,PASSWORD_SPO=<password_spo>,ENV=dev --config-file e2e_tests/cypress.config.ts --config baseUrl=https://make-recall-decision-dev.hmpps.service.justice.gov.uk
```
Comment (jackmaloney) [inline docs/tests.md:84]: Do we not need CRN's for the dev run?

File: .env.dev.sample (hunk 2 above)
Comment (jonnywyatt) [reply .env.dev.sample:12]: because that's not the right user for dev

File: docs/tests.md (hunk 3 above)
Comment (jonnywyatt) [reply docs/tests.md:84]: nope, the tests default to the correct ones

File: docs/tests.md (hunk 1 above)
Comment (jonnywyatt) [reply docs/tests.md:60]: fixed

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**